README.md

```diff
@@ -1,42 +1,39 @@
 # Openshift Nightlies
 
-This Repo defines Airflow Tasks used in running our nightly performance builds for stable and future releases of Openshift.
+This Repo defines Airflow DAGs used in running our nightly performance builds for stable and future releases of Openshift.
 
 
 ## Overview
 
-* `dags/openshift_nightlies` - Contains all Airflow code
+* `dags/openshift_nightlies` - Contains all Airflow code for Tasks as well as release configurations
 * `images` - Contains all custom images used in the Airflow DAGs
-* `charts` - Helm Charts for the Airflow Stack (includes Airflow, an EFK Stack for logging, Elastic/Kibana Cluster for results, and an instance of the perf-dashboard)
-* `scripts` - Install/Uninstall scripts for the Airflow stack
+* `charts` - Helm Charts for the PerfScale Stack (includes Airflow, an EFK Stack for logging, Elastic/Kibana Cluster for results, and an instance of the perf-dashboard)
+* `scripts` - Install/Uninstall scripts for the PerfScale Stack
 
-## Installing Airflow [Standalone]
 
-> Note: This requires you to have your own openshift cluster. If you wish to install this inside the Performance and Scale Baremetal Cluster (sailplane), please see the [Developer Playground](#installing-airflow-developer-playground)
+# Installation Methods
 
-To install Airflow you simply need to fork the repo and run the following on a box that has access to an openshift cluster:
+> All of these methods (minus the perfscale method which should only be used by perfscale team members) require you to fork this repo into your own user/organization. Please *DO NOT* attempt to install this by simply cloning the [upstream repo](https://github.com/cloud-bulldozer/airflow-kubernetes)
+
+## Tenant
+
+To install Airflow as a Tenant, you can run the following commands from the fork you wish to use as your tenant repo.
 
 ```bash
 # all commands are run at the root of your git repo
 # install the airflow stack and have it point to your fork of the dag code.
-# $PASSWORD refers to the password you would like to secure your airflow instance behind.
-./scripts/install.sh -p $PASSWORD
+# $PASSWORD refers to the password you would like to secure your airflow instance with.
+./scripts/tenant/create.sh -p $PASSWORD
```

Comment:
for the sake of consistency with Playground, why don't we call these `build.sh` and `cleanup.sh` ? `create` and `destroy` are fine but lack consistency.  @whitleykeith ?

Reply:
the main reason for naming them differently is to ensure people don't accidentally call the wrong script by using the wrong directory (which has been attempted multiple times with the standalone and playground modes).

medium term I want to add commands to the makefile to where someone does `make build playground` which will make the problem less persistent. At that point I'm not against making them the same name